pkg/cleaner/aws/spec.go

```diff
@@ -21,6 +22,10 @@ type CFClient interface {
 	UpdateTerminationProtection(*cloudformation.UpdateTerminationProtectionInput) (*cloudformation.UpdateTerminationProtectionOutput, error)
 }
 
+type Route53Client interface {
+	ListHostedZones(input *route53.ListHostedZonesInput) (*route53.ListHostedZonesOutput, error)
+}
```

Comment:
We do we need this interface?

Reply:
No idea. These have been there before for the other clients so I just aligned my changes with it.